### DISCUSSION TITLE
Reverse Captive Portal feature?

### DISCUSSION THREAD

**Discussion by RobertDuncan42** [RaspAP/raspap-webgui#780] [Q&A]
Not sure if I am requesting a new feature or just don't know how to do this.

Is it possible for the WiFi Client to be connected through a captive portal that requires an interaction before the link is established?

The one I am up against right now wants first name, last name and an email address. I have seen others that just want an email and some hotel rooms that ask for last name, room number.

Have others run into this while trying to connect the WiFi Client?

**Comment by ahoy** [comment]
It appears there is some functionality for being the client to a captive portal. See here: https://github.com/RaspAP/raspap-webgui/discussions/1039
